linkerd/proxy/http/src/detect.rs

```diff
@@ -92,6 +92,7 @@ where
     H: NewService<(HttpVersion, T)>,
 {
     pub fn new(target: T, server: Server, new_http: H, new_tcp: F, drain: drain::Watch) -> Self {
+        tracing::trace!("new AcceptHttp");
```

Comment:
nit/TIOLI: i put this in here while debugging, but i'm not sure if we really need it any more...
```suggestion
```